### PR TITLE
feat: move to @homex org and setup cicd

### PR DESCRIPTION
**Pull request by davideweaver**
* move to @homex org
* setup circleci and semantic-release

I'm not sure how this can be tested until i actually merge it to master.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          key: v1-npm-deps-{{ checksum "package-lock.json" }}
+          paths:
+            - node_modules
+jobs:
+  build_and_test:
+    docker:
+      - image: cimg/node:14.15
+    steps:
+      - checkout
+      - npm_ci
```
Comment (djdmbrwsk) [inline .circleci/config.yml:28]: Is this thing built in any way? Maybe with grunt? Should we have an npm script setup for that and then run it here?

**Review by djdmbrwsk — APPROVED** [review]
Looks good, one nitpick/question

**Comment by atdrago** [comment]
@davideweaver Do you mind also adding an `.nvmrc` that notes what version of Node this project uses?

File: .circleci/config.yml (hunk 1 above)
Comment (davideweaver) [reply .circleci/config.yml:28]: ah. `grunt` does do minifying/uglifying. I'll update. thanks

Comment (atdrago) [reply .circleci/config.yml:28]: Yeah, it's built using grunt

**Review by atdrago — APPROVED** [review]
Looks good!

**Pull request merged**